dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkerGroupServiceImpl.java

```diff
@@ -150,7 +153,7 @@ private boolean checkWorkerGroupNameExists(WorkerGroup workerGroup) {
      * @return boolean
      */
     private String checkWorkerGroupAddrList(WorkerGroup workerGroup) {
-        Map<String, String> serverMaps = zookeeperMonitor.getServerMaps(ZKNodeType.WORKER, true);
+        Map<String, String> serverMaps = registryMonitor.getServerMaps(NodeType.WORKER, true);
         for (String addr : workerGroup.getAddrList().split(Constants.COMMA)) {
```

Comment:
A little question, can we guarantee that addrList in each workergroup is not null?